src/tools/spectest-interp.cc

```diff
@@ -1274,15 +1274,21 @@ wabt::Result CommandRunner::Run(const Script& script) {
   for (const CommandPtr& command : script.commands) {
     switch (command->type) {
       case CommandType::Module:
-        OnModuleCommand(cast<ModuleCommand>(command.get()));
+        if (Failed(OnModuleCommand(cast<ModuleCommand>(command.get())))) {
+          PrintError(command->line, "invalid module");
+          return wabt::Result::Error;
```

Comment:
Why break out of the loop here?  Wouldn't you want to continue to try to run other tests?

Reply:
Sure, happy to do it that way -- it just requires rebasing all the test expectations. Done in latest commit.